tests/e2e/030_span_test.go

```diff
@@ -0,0 +1,186 @@
+		Name:    srcTapName,
+		Type:    vpp_interfaces.Interface_TAP,
+		Enabled: true,
+		IpAddresses: []string{
+			"10.10.1.2/24",
+		},
+		Link: &vpp_interfaces.Interface_Tap{
+			Tap: &vpp_interfaces.TapLink{
+				Version: 2,
+			},
+		},
+	}
+	srcLinuxTap := &linux_interfaces.Interface{
+		Name:    "linux_span_tap1",
+		Type:    linux_interfaces.Interface_TAP_TO_VPP,
+		Enabled: true,
+		IpAddresses: []string{
+			"10.10.1.1/24",
+		},
+		HostIfName: "linux_span_tap1",
+		Link: &linux_interfaces.Interface_Tap{
+			Tap: &linux_interfaces.TapLink{
+				VppTapIfName: srcTapName,
+			},
+		},
+	}
+
+	dstTap := &vpp_interfaces.Interface{
+		Name:    dstTapName,
+		Type:    vpp_interfaces.Interface_TAP,
+		Enabled: true,
+		IpAddresses: []string{
+			"10.20.1.2/24",
+		},
+		Link: &vpp_interfaces.Interface_Tap{
+			Tap: &vpp_interfaces.TapLink{
+				Version:        2,
+				ToMicroservice: fullMsName,
+			},
+		},
+	}
+	dstLinuxTap := &linux_interfaces.Interface{
+		Name:    "linux_span_tap2",
+		Type:    linux_interfaces.Interface_TAP_TO_VPP,
+		Enabled: true,
+		IpAddresses: []string{
+			"10.20.1.1/24",
+		},
+		HostIfName: "linux_span_tap2",
+		Link: &linux_interfaces.Interface_Tap{
+			Tap: &linux_interfaces.TapLink{
+				VppTapIfName: dstTapName,
+			},
+		},
+		Namespace: &linux_ns.NetNamespace{
+			Type:      linux_ns.NetNamespace_MICROSERVICE,
+			Reference: fullMsName,
+		},
+	}
+
+	spanRx := &vpp_interfaces.Span{
+		InterfaceFrom: srcTapName,
+		InterfaceTo:   dstTapName,
+		Direction:     vpp_interfaces.Span_RX,
+	}
+
+	ctx.startMicroservice(msName)
+	req := ctx.grpcClient.ChangeRequest()
+	err := req.Update(dstTap, dstLinuxTap, spanRx).Send(context.Background())
+	Expect(err).To(BeNil(), "Sending change request failed with err")
+
+	Eventually(ctx.getValueStateClb(dstTap), msUpdateTimeout).Should(
+		Equal(kvs.ValueState_CONFIGURED),
+		"Destination TAP is not configured",
+	)
+
+	Expect(ctx.getValueState(spanRx)).To(
+		Equal(kvs.ValueState_PENDING),
+		"SPAN is not in a `PENDING` state, but `InterfaceFrom` is not ready",
+	)
+
+	req = ctx.grpcClient.ChangeRequest()
+	err = req.Update(srcTap, srcLinuxTap).Send(context.Background())
+	Expect(err).To(BeNil())
+
+	Eventually(ctx.getValueStateClb(srcTap), msUpdateTimeout).Should(
+		Equal(kvs.ValueState_CONFIGURED),
+		"Source TAP is not configured",
+	)
+
+	Expect(ctx.getValueState(spanRx)).To(
+		Equal(kvs.ValueState_CONFIGURED),
+		"SPAN is not in a `CONFIGURED` state, but both interfaces are ready",
+	)
+
+	ctx.stopMicroservice(msName)
+	Eventually(ctx.getValueStateClb(dstTap), msUpdateTimeout).Should(
+		Equal(kvs.ValueState_PENDING),
+		"Destination TAP must be in a `PENDING` state, after its microservice stops",
+	)
+
+	Expect(ctx.getValueState(spanRx)).To(
+		Equal(kvs.ValueState_PENDING),
+		"SPAN is not in a `PENDING` state, but `InterfaceTo` is not ready",
+	)
+
+	// Check `show int span` output
+	var stdout bytes.Buffer
+	cmd := exec.Command("vppctl", "show", "int", "span")
```

Comment:
Perhaps add helper function `execVppctl(command string) (output string, err error)` (or more generally `execCommand` to make it usable even beyond vppctl) to `e2e_test.go`. The
```
var stdout bytes.Buffer
exec.Command(...)
cmd.Stdout = &stdout
...
```
sequence is repeated twice in this file, also occurs [here](https://github.com/ligato/vpp-agent/blob/dev/tests/e2e/e2e_test.go#L249-L254) and most likely will be needed in future tests.